docs/guides/troubleshooting.md

```diff
@@ -27,6 +27,16 @@ XCUITest driver offers a few methods to handle them.
     - When a permission alert exists on the top, it could select the `com.apple.springboard`
     - When another application is on the top by accepting/denying the system alert, or [`mobile: activateApp`](../reference/execute-methods.md#mobile-activateapp), the application would be selected as an active application.
 - [`mobile: alert`](../reference/execute-methods.md#mobile-alert)
+- `defaultActiveApplication` settings in [Settings](../reference/settings.md).
+    - `driver.settings.update({defaultActiveApplication: 'com.apple.springboard'})` (with [Appium Ruby client](https://github.com/appium/ruby_lib_core)) switches the current active application with `com.apple.springboard` instead of the given target application. It lets the session interact with the springboard. Please do not forget to set the value as `auto` like `driver.settings.update({defaultActiveApplication: 'auto'})` to bring the behavior back to the default behavior
```

Comment:
it does not switch, but rather changes the preferred application selection logic. If the setting value is "auto" then the preferred application (the one whose state is checked first) is always the application under test. Setting the value to a different bundle id simply shifts the previous validation, so the state of the given app is verified first.

Reply:
also springboard is a special case, because it's state is always "running in foreground", even if there are other apps running in foreground. that is why setting it as default will make WDA to always select it as active.

Reply:
Yea. Maybe just a link to settings would be nice to avoid duplication explanation